tools/cli/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "jetpack-cli",
+	"name": "@automattic/jetpack-cli",
```

Comment:
I note this collides with https://github.com/Automattic/jetpack/blob/0459c528543015df4deea8e5782cc1def984f68d/projects/js-packages/jetpack-cli/package.json#L2

I'm also surprised it doesn't seem to have broken https://github.com/Automattic/jetpack/blob/0459c528543015df4deea8e5782cc1def984f68d/package.json#L33

Reply:
Addressed in c3ae39b.